### PR TITLE
People Search Larger Images Toggle

### PR DESCRIPTION
**Pull request by Davideo37**
Added a checkbox for people search on desktop that enables larger avatar previews for the results list. This should close issue #1212.

before 
Screen Shot 2021-06-25 at 4 36 58 PM

After 
image

### CODE REVIEW

**Comment by roddy3** [comment]
worked closely with David on this one. someone else needs to review it.

**Comment by Davideo37** [comment]
Updated UI with the Advanced Search "Include:" boxes

Before selection:
ui
Normal search results without Large Images selected:
ui2
Search Results with Large Images selected:
ui3
ui4

File: src/views/PeopleSearch/components/PeopleSearchResult/index.js [hunk 1]
```diff
@@ -160,7 +203,7 @@ export default class PeopleSearchResult extends Component {
             >
               <Grid item xs={1}>
                 <IMG
-                  className="people-search-avatar"
+                  className={'people-search-avatar'}
```
Comment (silasw2) [inline src/views/PeopleSearch/components/PeopleSearchResult/index.js:206]: Any particular reason you changed this here? From what I know wouldn't the previous way have worked?

**Review by silasw2 — APPROVED** [review]
LGTM! Left a small comment and maybe wait til tomorrow to merge so you can get one more reviewer.

File: src/views/PeopleSearch/components/PeopleSearchResult/index.js (hunk 1 above)
Comment (Davideo37) [reply src/views/PeopleSearch/components/PeopleSearchResult/index.js:206]: Might've been a 'Prettier' modification. I can undo it

File: src/views/PeopleSearch/index.js [hunk 2]
```diff
@@ -90,55 +90,51 @@ const noResultsCard = (
...
+      <Grid item>
+        <Typography variant="body2" style={styles.headerStyle}>
+          RESULTS
+        </Typography>
+      </Grid>
+    </Grid>
+  </div>
 );
 
 class PeopleSearch extends Component {
```
Comment (josh-peters-2357) [inline src/views/PeopleSearch/index.js:140]: I think we should refactor this to a functional component while we're touching these files.

File: src/views/PeopleSearch/index.js [hunk 3]
```diff
@@ -560,11 +577,29 @@ class PeopleSearch extends Component {
...
+          <Media
+            query="(min-width: 960px)"
+            render={() => (
+              <Grid item xs={12} align="center">
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.displayLargeImage}
+                      onChange={() => {
+                        this.handleChangeDisplayLargeImages();
```
Comment (josh-peters-2357) [inline src/views/PeopleSearch/index.js:589]:
```suggestion
                        this.setState({displayLargeImage: !this.state.displayLargeImage});
                        this.search();
```
This function is only called one time, and it happens to be here. It will only ever be called one time because of what it's doing, so we can put the two lines it contained in the onClick.

**Review by josh-peters-2357 — CHANGES_REQUESTED** [review]
Looks pretty good! I left a couple comments, but I would also suggest that while we're here, we use the network hook we designed instead of what's currently here.

File: src/views/PeopleSearch/index.js (hunk 2 above)
Comment (silasw2) [reply src/views/PeopleSearch/index.js:140]: This is a smart line of thinking, however, even Cam was hesitant to begin that task, at least now. For the future I think this is a good goal, but these files are quite complicated when it comes to the react tools they use so converting it to a functional component would be a longer process than we have time for currently.

File: src/views/PeopleSearch/index.js (hunk 3 above)
Comment (Davideo37) [reply src/views/PeopleSearch/index.js:589]: I realize the function is only 2 lines, but so are half the other methods for handling changes in the search parameters. I was mirroring their structure which keeps the code in the JSX as clean as possible. I can still change this but I feel like even though it's only 2 lines, it would be better practice to stay consistent.

File: src/views/PeopleSearch/components/PeopleSearchResult/index.js [hunk 4]
```diff
@@ -143,6 +143,49 @@ export default class PeopleSearchResult extends Component {
...
+                />
+              </Grid>
+              <Grid
+                item
+                style={{
+                  // a set width is necessary to keep profile images in line
+                  // while maintaining center alignment
+                  width: '260px',
+                }}
+              >
```
Comment (EjPlatzer) [inline src/views/PeopleSearch/components/PeopleSearchResult/index.js:177]: There's an easier and better way to keep the images centered without setting a static width:
```suggestion
              <Grid item xs={6} container justify="flex-end">
                <IMG
                  className="people-search-avatar-large"
                  src={`data:image/jpg;base64,${this.state.avatar}`}
                  alt=""
                  noLazyLoad="true"
                  placeholderColor="#eeeeee"
                />
              </Grid>
              <Grid item xs={6}>
```

**Review by EjPlatzer — CHANGES_REQUESTED** [review]
Given the time pressure we're under, this seems like a good short-term solution. I think we'll need to re-examine the search options and how to best lay them out on the search form in the long run. 

This is well-coded and works nicely - great job! I do have one suggestion, but otherwise it's coming along nicely.

File: src/views/PeopleSearch/components/PeopleSearchResult/index.js (hunk 4 above)
Comment (Davideo37) [reply src/views/PeopleSearch/components/PeopleSearchResult/index.js]: I didn't catch that part when I was copying from the mobile view, thanks for finding that!

**Review by EjPlatzer — DISMISSED** [review]
LGTM!

File: src/views/PeopleSearch/index.js (hunk 3 above)
Comment (josh-peters-2357) [reply src/views/PeopleSearch/index.js:589]: Well, consistency is good. Discreteness is good too. Although, I like efficiency and I don't like reusability when it won't get reused. But it is a difference of philosophy so I'm just gonna resolve it.

**Review by josh-peters-2357 — APPROVED** [review]

File: src/views/PeopleSearch/index.js [hunk 5]
```diff
@@ -118,7 +118,7 @@ const peopleSearchHeaderDesktop = (
         <Typography variant="body2" style={styles.headerStyle}>
           @GORDON.EDU
           <br />
-          MAIL LOCATION
+          MAILBOX #
```
Comment (EjPlatzer) [inline src/views/PeopleSearch/index.js:121]: I'm not sure why this change was made, but it will be confusing for Fac/Staff results now because we have mailstops instead of mailboxes. For example, my mailstop in prod is CTS, so `Mailbox #CTS` doesn't make much sense.
